### PR TITLE
Feature/responsive nav

### PR DESCRIPTION
**Pull request by mlralcaide**
This pull request features adding a Logo 'Loruki.' to the responsive nav as well putting fixed the nav bar and little changes in background colors on the links hover.

@ajonesb please check the pull request.

**Nav Bar fixed:**
image

**Hover with another background color:**
image

**Changed height, from 100vh to auto:**
image

### CODE REVIEW

File: css/navigation.css [hunk 1]
```diff
@@ -48,13 +57,13 @@
   .nav-area ul {
     position: fixed;
     width: 100%;
-    height: 100vh;
-    background: rgb(39, 39, 39);
+    height: auto;
```
Comment (ajonesb) [inline css/navigation.css]: Please make this height 100vh in order to fill up the whole screen. It ca't be halfway the screen.

File: css/navigation.css [hunk 2]
```diff
@@ -66,7 +75,7 @@
     font-size: 30px;
   }
   .nav-area ul li:hover {
-    background: #000;
+    background: var(--dark-color);
```
Comment (ajonesb) [inline css/navigation.css:78]: Cool good catch!

File: css/navigation.css [hunk 3]
```diff
@@ -75,3 +84,7 @@
     left: 0;
   }
 }
+
+/* Logo */
```
Comment (ajonesb) [inline css/navigation.css]: Please remove unneccessary comment at the bottom of the navigation.css file.

**Comment by mlralcaide** [comment]
Reaty to merge.

**Comment by mlralcaide** [comment]
Cancele el pull request para poder hacer el merge, pero creo que no lo he logrado, quice hacer un nuevo pull request, asi que reabri este.

**En la imagen puedes ver que hice el merce de feature/responsive-nav a master, no se si estara bien:**
image

**Pull request merged**